src/EventStore.Core.Tests/ClientOperations/when_starting_a_transaction_expecting_verion_any.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
```

Comment:
Fix spelling of `version` in the filename.

Reply:
fixed